core/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -297,17 +297,17 @@ public Query toQuery(QueryParseContext parseContext) throws QueryParsingExceptio
             return new MatchAllDocsQuery();
         }
 
-        BooleanQuery booleanQuery = new BooleanQuery(this.disableCoord);
-        addBooleanClauses(parseContext, booleanQuery, this.mustClauses, BooleanClause.Occur.MUST);
-        addBooleanClauses(parseContext, booleanQuery, this.mustNotClauses, BooleanClause.Occur.MUST_NOT);
-        addBooleanClauses(parseContext, booleanQuery, this.shouldClauses, BooleanClause.Occur.SHOULD);
-        addBooleanClauses(parseContext, booleanQuery, this.filterClauses, BooleanClause.Occur.FILTER);
-
-        booleanQuery.setBoost(this.boost);
-        Queries.applyMinimumShouldMatch(booleanQuery, this.minimumShouldMatch);
-        Query query = this.adjustPureNegative ? fixNegativeQueryIfNeeded(booleanQuery) : booleanQuery;
-        if (this.queryName != null) {
-            parseContext.addNamedQuery(this.queryName, query);
```

Comment:
I found the use of it inconsistent in most methods. I left it in the getters/setters where it makes sense to distinguish between method parameter and class member with the same name. I think I heard there's no real style guide around this, but I wanted to avoid having both variants in the same method at least.